src/elements/Button/ButtonTheme.ts

```diff
@@ -77,7 +77,7 @@ const baseTheme: Partial<ButtonTheme> = {
     }
   },
   sizes: {
-    small: 'text-xs px-5px py-0.5 leading-[normal]',
+    small: 'text-sm px-5px py-0.5 leading-[normal]',
```

Comment:
Let's just be cautious changing these too much - not sure what other composite components ( think button in select/etc ) it could affect.

Reply:
yea makes sense, there is no `text-xs` in our tw config is why I changed this. So it was applying the base text size